models/public/vitstr-small-patch16-224/accuracy-check.yml

```diff
@@ -0,0 +1,303 @@
+models:
+  - name: vitstr-small-patch16-224-onnx
+    launchers:
+      - framework: onnx_runtime
+        model: vitstr-small-patch16-224.onnx
+        adapter:
+          type: simple_decoder
+          start_index: 1
+          do_lower: True
+          custom_label_map:
```

Comment:
could you please implement vocab.txt based initialization for label map in adapter?